Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/trees/Tree.tsx

```diff
@@ -1,97 +1,88 @@
 import React, { useCallback, useEffect, useState } from 'react';
```

Comment:
The whole logic of this component is duplicated in `CategoryTree`. This is incorrect..